l1-contracts/contracts/transactionFilterer/GatewayTransactionFilterer.sol

```diff
@@ -5,11 +5,17 @@ pragma solidity 0.8.24;
 import {Ownable2StepUpgradeable} from "@openzeppelin/contracts-upgradeable-v4/access/Ownable2StepUpgradeable.sol";
 
 import {AlreadyWhitelisted, InvalidSelector, NotWhitelisted, ZeroAddress} from "../common/L1ContractErrors.sol";
+import {L2_ASSET_ROUTER_ADDR} from "../common/L2ContractAddresses.sol";
 import {ITransactionFilterer} from "../state-transition/chain-interfaces/ITransactionFilterer.sol";
 import {IBridgehub} from "../bridgehub/IBridgehub.sol";
 import {IAssetRouterBase} from "../bridge/asset-router/IAssetRouterBase.sol";
 import {IL2AssetRouter} from "../bridge/asset-router/IL2AssetRouter.sol";
 
+/// @dev We want to ensure that only whitelisted contracts can ever be deployed,
+/// while allowing anyone to call any other method. Thus, we disallow calls that can deploy contracts
+/// (i.e. calls to the predeployed Create2Factory or ContractDeployer).
+address constant MIN_ALLOWED_ADDRESS = address(0x20000);
```

Comment:
would it make sense to move this to one of the config files, maybe L2ContractAddresses?

Reply:
This address is only used here + I wanted to change as little contract hashes as possible